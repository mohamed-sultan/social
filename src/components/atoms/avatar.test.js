import Avatar from './avatar';
import {cleanup, render} from 'react-native-testing-library';

describe('Home', () => {
  afterEach(cleanup); //Unmounts React trees that were mounted with render to prevent memory leak.
  it('should render correctly', () => {
    const {toJSON} = render( <Avatar uri='' />);
    expect(toJSON()).toMatchSnapshot(); 
  });
});