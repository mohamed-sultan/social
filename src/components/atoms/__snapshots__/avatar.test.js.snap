// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render correctly 1`] = `
<Image
  source={
    {
      "uri": "",
    }
  }
  style={
    {
      "borderRadius": 30,
      "height": 60,
      "width": 60,
    }
  }
/>
`;
